spec/features/user_can_searc_for_a_specific_zipcode_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe "zipcode search" do
+  it "user can search for events by a specific zipcode" do
+
+    visit root_path
+
+    fill_in "q", with: "80202"
+    click_on "Find Events"
+    save_and_open_page
+    expect(current_path).to eq("/search")
+    expect(page).to have_content("16 events for March")
+
+    events = JSON.parse(response.body)
```

Comment:
so because we're testing the output of our page, which in this case is the HTML we're seeing with the event details, we won't need to parse our body here. we want to parse our response from our Faraday request up in our `EventsService`